core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -199,6 +202,43 @@ angular.module('oppia').directive('adminMiscTab', [
           );
         };
 
+        ctrl.getModelsRelatedToUser = function() {
+          ctrl.setStatusMessage('Getting the models related to user...');
+          $http.get(GET_MODELS_RELATED_TO_USER_HANDLER_URL, {
+            params: { user_id: ctrl.userIdToGet }
+          }).then(
+            function(response) {
+              if (response.data.related_models_exist) {
+                ctrl.setStatusMessage(
+                  'Some related models exist, see logs ' +
+                  'to find out the exact models'
+                );
+              } else {
+                ctrl.setStatusMessage('No related models exist');
+              }
+            },
+            function(errorResponse) {
+              ctrl.setStatusMessage(
+                'Server error: ' + errorResponse.data.error);
+            }
+          );
+        };
+
+        ctrl.deleteUser = function() {
+          ctrl.setStatusMessage('Starting the deletion of the user...');
```

Comment:
Maybe add a confirmation (is this irreversible?)

Reply:
I agree with this, if possible let's have a confirmation modal for this. (OR ask the username twice in the form)